Puppetfile

```diff
@@ -1,12 +1,9 @@
 forge 'https://forgeapi.puppetlabs.com'
 
-mod 'katello/candlepin',             :git => 'https://github.com/theforeman/puppet-candlepin'
-mod 'katello/foreman_proxy_content', :git => 'https://github.com/theforeman/puppet-foreman_proxy_content'
-mod 'katello/certs',                 :git => 'https://github.com/theforeman/puppet-certs'
-mod 'katello/katello',               :git => 'https://github.com/theforeman/puppet-katello'
-mod 'katello/pulp',                  :git => 'https://github.com/theforeman/puppet-pulp'
-mod 'katello/qpid',                  :git => 'https://github.com/theforeman/puppet-qpid'
-
-# A fork that marks metadata as compatible with puppet 5
-# https://github.com/thias/puppet-squid3/pull/59
-mod 'thias-squid3',                  :git => 'https://github.com/theforeman/puppet-squid3', :ref => 'allow-puppet-5'
+mod 'katello/candlepin',              '>= 5.1.1 < 5.2.0'
+mod 'katello/foreman_proxy_content',  '>= 7.2.0 < 7.3.0'
+mod 'katello/certs',                  '>= 4.2.0 < 4.3.0'
+mod 'katello/katello',                '>= 8.0.1 < 8.1.0'
+mod 'katello/pulp',                   '>= 5.5.0 < 5.6.0'
+mod 'katello/qpid',                   '>= 4.2.2 < 4.3.0'
+mod 'thias-squid3',                   :git => 'https://github.com/theforeman/puppet-squid3', :ref => 'allow-puppet-5'
```

Comment:
I'd appreciate keeping the comment for this odd one in. While you're at it, could you decreasing indenting by 1 so it matches the previous indenting?